frontend/src/main/java/com/google/tripmeout/frontend/servlet/TripServlet.java

```diff
@@ -0,0 +1,58 @@
+package com.google.tripmeout.frontend.servlet;
+
+import com.google.gson.Gson;
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import com.google.tripmeout.frontend.TripModel;
+import com.google.tripmeout.frontend.error.TripMeOutException;
+import com.google.tripmeout.frontend.error.TripNotFoundException;
+import com.google.tripmeout.frontend.storage.InMemoryTripModelStorage;
+import java.io.IOException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@Singleton
+public final class TripServlet extends HttpServlet {
+  // The Trip get and delete servlet
+  private static final long serialVersionUID = 1L;
+  private final InMemoryTripModelStorage storage;
+  private static final String APPLICATION_JSON_CONTENT_TYPE = "application/json";
```

Comment:
Can you reorder this?

Generally we like to keep our fields in groups. You usually want statics first. Overall most java code I've seen at large companies (like ours) uses this order for groups (if they exist) with spaces in between:
1) Logger
1) Non-logger private static finals (private constants)
1) Public static finals (public constants)
1) Private finals (injected fields)
1) Private non-finals (mutable instance data)

```java
class ExampleLister {
  private static final GoogleLogger logger = GoogleLogger.forEnclosingClass();

  private static final long MAX_PAGE_SIZE = 1_000L;

  /** Used to indicate listing in descending order. */
  public static final String DESC_LIST_ORDER_KEY = "desc";
  /** Used to indicate listing in ascending order. */
  public static final String ASC_LIST_ORDER_KEY = "asc";

  private final ExampleStorage;

  public ExampleLister(ExampleStorage storage) { ... }
}
```